website/content/specification/syntax/constraints.md

```diff
@@ -164,6 +164,21 @@ And the following document.
 The expect constraint would pass for each `sibling` in the `parent` named "p1", and would fail for each `sibling` in the `parent` named "p2".
 
 
+## `is-unique` constraints
+----
```

Comment:
This is looking good but can you removed the dashes?

```suggestion
```

Reply:
Removed